From 30beb4887e26b7ef1204dad38f6577e5c635ef52 Mon Sep 17 00:00:00 2001
From: Alex Deryskyba <alex@codesnake.com>
Date: Mon, 8 Sep 2014 23:29:40 +0300
Subject: [PATCH 08/15] Reorder libraries in configure script to prevent
 linker errors when linking with libsmbclient

Place libsmbclient before all other libraries to prevent linker errors when linking
with libsmbclient if the libc that is currently used doesn't contain some functions
such as dn_expand (which are often included in libc), but are actually included in
libresolv.
---
 configure.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 2e49fa1..d1dd1ce 100644
--- a/configure.in
+++ b/configure.in
@@ -1561,7 +1561,7 @@ fi
 # samba
 if test "x$use_samba" != "xno"; then
   PKG_CHECK_MODULES([SAMBA], [smbclient],
-    [INCLUDES="$INCLUDES $SAMBA_CFLAGS"; LIBS="$LIBS $SAMBA_LIBS"],
+    [INCLUDES="$INCLUDES $SAMBA_CFLAGS"; LIBS="$SAMBA_LIBS $LIBS"],
     [AC_CHECK_LIB([smbclient], [main],,
       use_samba=no;AC_MSG_ERROR($missing_library))
       USE_LIBSMBCLIENT=0
-- 
1.7.9.5

